tools/run_tests/run_xds_tests.py

```diff
@@ -2238,6 +2238,35 @@ def test_csds(gcp, original_backend_service, instance_group, server_uri):
                        test_csds_timeout_s)
 
 
+def maybe_write_sponge_properties():
+    """Writing test infos to enable more advanced testgrid searches."""
+    if 'KOKORO_ARTIFACTS_DIR' not in os.environ:
+        return
+
+    project_root = os.path.join(os.path.dirname(os.path.abspath(__file__)),
+                                '../..')
+    git_origin_url = subprocess.getoutput('git -C "%s" remote get-url origin' %
+                                          project_root)
+    git_commit_short = subprocess.getoutput(
+        'git -C "%s" rev-parse --short HEAD' % project_root)
+
+    properties = [
+        # Technically, 'TESTS_FORMAT_VERSION' is not required for run_xds_tests.
+        # We keep it here so one day we may merge the process of writing sponge
+        # properties.
+        'TESTS_FORMAT_VERSION,2',
+        'TESTGRID_EXCLUDE,%s' % os.environ.get('TESTGRID_EXCLUDE', 0),
+        'GIT_ORIGIN_URL,%s' % git_origin_url,
```

Comment:
The output from your example doesn't seem right: `GIT_ORIGIN_URL,/var/local/jenkins/grpc`
This should be something like `GIT_ORIGIN_URL,https://github.com/lidizheng/grpc.git`

Reply:
🤦  why we clone in this way...

```
+ mkdir -p /var/local/git
+ git clone /var/local/jenkins/grpc /var/local/git/grpc
```

I will find another way to pass this info through.

Reply:
The good news is the properties were successfully picked up by the job! https://fusion2.corp.google.com/invocations/c01fc70b-628a-4260-a11e-696ae13155b5/details